hwproj.front/src/components/Homeworks/AddHomework.tsx

```diff
@@ -130,18 +150,31 @@ export default class AddHomework extends React.Component<IAddHomeworkProps,
                                                 onChange={(e) => (task.maxRating = +e.target.value)}
                                             />
                                         </Grid>
-                                        <Grid>
+                                        <Tabs
+                                            onChange={(event, newValue) => (this.onUpdateIsPreviewInTask(index, newValue === 1))}
+                                            indicatorColor="primary"
+                                            value={task.isPreview ? 1 : 0}
+                                        >
+                                            <Tab label="Редактировать"/>
+                                            <Tab label="Превью"/>
+                                        </Tabs>
+
+                                        <div role="tabpanel" hidden={task.isPreview}>
                                             <TextField
                                                 multiline
                                                 fullWidth
-                                                rows="10"
-                                                label="Условие задачи"
+                                                rows="4"
+                                                rowsMax="20"
+                                                label="Описание"
                                                 variant="outlined"
                                                 margin="normal"
                                                 name={task.description}
                                                 onChange={(e) => (task.description = e.target.value)}
                                             />
-                                        </Grid>
+                                        </div>
+                                        <div role="tabpanel" hidden={!task.isPreview}>
```

Comment:
можно вообще не рендерить этот элемемент

```suggestion
                                        task.isPreview && <div role="tabpanel">
```